src/pybind/mgr/dashboard/frontend/src/app/shared/components/config-option/config-option.component.scss

```
@@ -0,0 +1,24 @@
+hr {
+  margin-top: 5px;
+  margin-bottom: 5px;
+}
+
+.control-label {
+  text-align: left;
+}
+
+.checkbox-primary {
+  input {
+    width: 23px;
+    height: 15px;
+    margin-left: 0;
+    cursor: pointer;
+  }
+  label {
+    &:before,
+    &:after {
+      margin-left: 0;
+    }
+    cursor: auto;
+  }
+}
```

The input on the right is very close to the separation line, this will put it on the same level as the text.
```suggestion
}

.form-group {
  .col-sm-6 {
    padding-top: 7px;
  }
}
```